server/controllers/stock/core.js

```diff
@@ -473,8 +497,10 @@ function computeInventoryIndicators(inventories) {
   for (let i = 0; i < inventories.length; i++) {
     const inventory = inventories[i];
 
-    // the quantity of stock available in the given depot
-    const Q = inventory.quantity; // the quantity
+    const inventoryExpiredLotsQuantity = inventory.expiredLotsQuantity || 0;
+
+    const Q = inventoryExpiredLotsQuantity
+      ? inventory.quantity - inventoryExpiredLotsQuantity : inventory.quantity; // the quantity
```

Comment:
@mbayopanda  The variable **enableExpiredStockOut** I used expiredLotsQuantity comes from this function, it's a bit confusing since I reused the same name for the property
````
  // verification of the activation of the option enable expired stock_out,
  // search for expired quantities in order to calculate the stock management
  // indicators with the usable quantities
  if (stockParams && stockParams.enable_expired_stock_out) {
    const lotsAvailable = await getLotsDepot(null, stockParams);

    lots.forEach(inventory => {
      inventory.expiredLotsQuantity = 0;

      lotsAvailable.forEach(lot => {
        const hasSameDepot = lot.depot_uuid === inventory.depot_uuid;
        const hasSameInventory = lot.inventory_uuid === inventory.inventory_uuid;

        if (hasSameDepot && hasSameInventory) {
          if (lot.expired) {
            inventory.expiredLotsQuantity += lot.quantity;
          }
        }
      });
    });
  }
````
in server/controllers/stock/core.js  line **359**